<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewerClass</class>
 <widget class="QMainWindow" name="ViewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Viewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="centralLayout" stretch="2,6,2">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="leftLayout" stretch="3,2">
      <item>
       <widget class="QTabWidget" name="tabWidget1">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="Page1" native="true">
         <attribute name="title">
          <string>显示</string>
         </attribute>
         <layout class="QGridLayout" name="propertyLayout" columnstretch="1,3">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item row="9" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="colorGLabel">
            <property name="text">
             <string>G：</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSlider" name="alpha">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSlider" name="colorB">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="colorBLabel">
            <property name="text">
             <string>B：</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="windowCenterLabel">
            <property name="text">
             <string>窗位：</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <layout class="QHBoxLayout" name="windowWidthLayout" stretch="5,1">
            <item>
             <widget class="QSlider" name="windowWidth">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>1000</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
              <property name="pageStep">
               <number>100</number>
              </property>
              <property name="value">
               <number>800</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="windowWidthVal">
              <property name="text">
               <string>800</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="6" column="1">
           <layout class="QHBoxLayout" name="windowCenterLayout" stretch="5,1">
            <item>
             <widget class="QSlider" name="windowCenter">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>1000</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
              <property name="pageStep">
               <number>100</number>
              </property>
              <property name="value">
               <number>200</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="windowCenterVal">
              <property name="text">
               <string>200</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="3" column="1">
           <widget class="QSlider" name="colorG">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="layerNameLabel">
            <property name="text">
             <string>图层：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="layerName"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="windowWidthLabel">
            <property name="text">
             <string>窗宽：</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSlider" name="colorR">
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>255</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="colorRLabel">
            <property name="text">
             <string>R：</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="alphaLabel">
            <property name="text">
             <string>透明度：</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="isoValueLabel">
            <property name="text">
             <string>等值面</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <layout class="QHBoxLayout" name="isoValueLayout" stretch="5,1">
            <item>
             <widget class="QSlider" name="isoValue">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>1000</number>
              </property>
              <property name="singleStep">
               <number>10</number>
              </property>
              <property name="pageStep">
               <number>100</number>
              </property>
              <property name="value">
               <number>200</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="isoValueVal">
              <property name="text">
               <string>200</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="Page2">
         <attribute name="title">
          <string>变形</string>
         </attribute>
         <layout class="QGridLayout" name="transformLayout" columnstretch="1,3">
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="layerName1">
            <property name="text">
             <string>图层：</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="layerNameLabel1"/>
          </item>
          <item row="1" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="settingLayout">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab1">
           <attribute name="title">
            <string>显示图层</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QTableWidget" name="layerTable"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab2">
           <attribute name="title">
            <string>绘制属性</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>5</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="LineLayout1" stretch="1,3">
              <item>
               <widget class="QLabel" name="renderLabel">
                <property name="text">
                 <string>绘制方式</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="renderComboBox">
                <item>
                 <property name="text">
                  <string>面绘制</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>体绘制</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="LineLayout2" stretch="1,3">
              <item>
               <widget class="QLabel" name="ambientLabel">
                <property name="text">
                 <string>环境光</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="ambientSlider">
                <property name="value">
                 <number>0</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="LineLayout3" stretch="1,3">
              <item>
               <widget class="QLabel" name="diffuseLabel">
                <property name="text">
                 <string>漫反射</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="diffuseSlider">
                <property name="value">
                 <number>80</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="LineLayout4" stretch="1,3">
              <item>
               <widget class="QLabel" name="SpecularLabel">
                <property name="text">
                 <string>镜面反射</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="SpecularSlider">
                <property name="value">
                 <number>20</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="LineLayout5">
              <item>
               <widget class="QCheckBox" name="AxesCheckBox">
                <property name="text">
                 <string>显示坐标轴</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="SliceCheckBox">
                <property name="text">
                 <string>显示3D切片</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="mainLayout"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="sliceLayout" stretch="1,1,1">
      <property name="spacing">
       <number>1</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="sliceLayout1">
        <property name="spacing">
         <number>0</number>
        </property>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="sliceLayout2">
        <property name="spacing">
         <number>0</number>
        </property>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="sliceLayout3">
        <property name="spacing">
         <number>0</number>
        </property>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpenDicom"/>
    <addaction name="actionOpenNifti"/>
    <addaction name="actionSaveNifti"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpenDicom">
   <property name="text">
    <string>打开Dicom</string>
   </property>
  </action>
  <action name="actionOpenNifti">
   <property name="text">
    <string>打开Nifti</string>
   </property>
  </action>
  <action name="actionSaveNifti">
   <property name="text">
    <string>导出Nifti</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Viewer.qrc"/>
 </resources>
 <connections/>
</ui>
